build.gradle

```diff
@@ -49,15 +49,23 @@ allprojects {
         }
 
         maven {
-            url = 'https://maven.wso2.org/nexus/content/repositories/orgballerinalang-1614'
+            url = 'https://repo.maven.apache.org/maven2'
         }
 
         maven {
-            url = 'https://repo.maven.apache.org/maven2'
+            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
         }
 
         maven {
             url = 'https://maven.pkg.github.com/ballerina-platform/module-ballerina-io'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
         }
```

Comment:
Any reason to add this repo as dependency repository?

Reply:
Don't we need these credentials to publish the package?

Reply:
No the publishing configs are given in under `io-ballerina` build gradle

Reply:
Thanks, will send another PR to remove it.